src/components/tabs/Tabs.module.css

```diff
@@ -7,19 +7,49 @@
 	margin-inline-start: -1rem;
 	overflow-x: scroll;
 	/* Hide scrollbar on firefox */
-	scrollbar-width: none; 
+	scrollbar-width: none;
 }
 
 .tab-scroll-overflow::-webkit-scrollbar {
 	/* Hide scrollbar on chrome / safari */
-  display: none;
+	display: none;
 }
 
-.tab:focus {
+.tab,
+.tab:focus,
+.tab:active {
+	border-bottom: 0 !important;
+}
+
+.tab:focus-visible {
 	outline: 2px solid var(--theme-accent);
 	outline-offset: -2px;
 }
 
 .tabpanel {
 	padding-block-start: 1rem;
 }
+
+.tablist {
+	position: relative;
+}
+
+.selectedIndicator {
+	position: absolute;
+	bottom: -4px;
+	height: 4px;
+	background-color: var(--theme-accent);
+	transition-duration: 200ms;
```

Comment:
It would be better to move this behind a `prefers-reduced-motion: no-preference` query. That way we default to no transition and turn on the transition for users who are OK with it.

```suggestion
```

Reply:
Alright, this seems fine, but I'm not sure that this really changes anything. The end result will always end up being the same, no matter which way we structure the CSS, right?